cleanlab/datalab/internal/spurious_correlations.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np 
+import pandas as pd
+from sklearn.model_selection import cross_val_predict, cross_val_score
+from sklearn.naive_bayes import GaussianNB
+from statistics import mode
+import warnings 
+warnings.filterwarnings('ignore')
+
+from datalab import DataLab
+
+class SpuriousCorrelations:
+
+    def __init__(self, data: DataLab) -> None:
+        self.data = data
+        self.issues = data.issues
+        self.labels = data.labels  
+
+    def spurious_correlations(self) -> pd.DataFrame:
+        baseline_accuracy = np.bincount(self.labels).argmax() / len(self.labels)
+        image_properties = ["near_duplicate_score", "blurry_score", "light_score", "low_information_score", "dark_score", "grayscale_score", "odd_aspect_ratio_score", "odd_size_score"]
```

Comment:
before you loop over these, you should restrict to only the subset of these that is present in: `self.issues`

In some cases, not all of these properties will have been previously computed, in which case we should just not compute the spurious correlation for those properties that were not computed already